contracts/src/core/interfaces/IEigenDAThresholdRegistry.sol

```diff
@@ -44,6 +44,10 @@ interface IEigenDAThresholdRegistry {
 
     ///////////////////////// V2 ///////////////////////////////
 
+    /// @notice Returns the next blob version
+    /// @dev Can be called before calling getBlobParams to verify that an input blobVersion actually exists
+    function nextBlobVersion() external view returns (uint16);
```

Comment:
is this getting called anywhere?

Reply:
ah I see:
```
        uint16 nextBlobVersion = eigenDAThresholdRegistry.nextBlobVersion();
        if (blobVersion >= nextBlobVersion) {
            revert InvalidBlobVersion(blobVersion, nextBlobVersion);
        }
```

so what happens to the cert validity when `nextBlobVersion+=1`? If it was once failing on this check and the blob version updates, could it pass?

Reply:
I don't this changing matters for zk proving but for optimistic rollups it might